src/platforms/java/guides/spring-boot/async.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Async Methods
```

Comment:
is it exactly the same as src/platforms/java/guides/spring/async.mdx ?
What's about using the `includes` folder or linking to the other page? duplicating everything is always a bit hard to maintain I guess, not sure how much we'd have to maintain this

Reply:
Given that these are so similar, yes, I think it makes sense to make this a piece of common content for Java (so it would live in the folder **src > platforms > Java > common**. Then add includes for the parts that differ between the two. And you can add a "not supported" component in the front matter so that it doesn't show up for the other Java frameworks that don't use this.